tests/fixtures/MailsMockTraitTest/.gitignore

```diff
@@ -0,0 +1 @@
+test_mail_with_export.html
```

Comment:
?

Reply:
It is a temporary file which we creating in `testMailWithExport`.